specs/engage-digital_openapi3.yaml

```diff
@@ -2956,6 +2956,10 @@ paths:
               schema:
                 $ref: '#/components/schemas/IdentityGroup'
           description: Success
+        '409':
+          description: >-
+            If the Identity Group is already being updated, it will return a 409
+            error.

```

Comment:
why the newline ?

Reply:
specs/engage-digital_openapi3.yaml:3411 used as a reference, honestly not much more thought than that went into it.